### ISSUE TITLE
READMEから住所コード記載の削除

### ISSUE THREAD

**Issue by nyampire** [yuuhayashi/citygml-osm#68]
`住所コード` は変換の対象ではなくなった (#46 ) ので、READMEから記述を削除したいです。

**Comment by yuuhayashi** [comment]
2022-01-30 時点のREADME.md で既に "addr:ref" は削除されているはずです。

**Comment by nyampire** [comment]
はい、`addr:ref`の記述は無くなっているのですが、表のなかに、住所コードに関する記載が残っているので、以下の行を消したい、という意図です。

https://github.com/yuuhayashi/citygml-osm/blob/69c0510a089ae42fb8ed7f713be2b7810578421c/README.md?plain=1#L287
